system/src/main_passive.cpp

```diff
@@ -168,10 +168,6 @@ void app_setup_and_loop_passive(void)
             DECLARE_SYS_HEALTH(RAN_Loop);
 //            DEBUG("Exited User Loop");
         }
-
-        if (!SPARK_FLASH_UPDATE) {
```

Comment:
Yikes! I must have missed that in my previous merge, somehow. NOTHING in main_passive.cpp should be changing.